pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -324,6 +338,27 @@ func ImageFromNative(n *native.Image) (*Image, error) {
 	i.Size = n.Size
 	return i, nil
 }
+
+// mountsFromNative only filters bind mount to transform from native container.
+// Because native container shows all types of mounts, such as tmpfs, proc, sysfs.
+func mountsFromNative(spMounts []specs.Mount) []MountPoint {
+	mountpoints := make([]MountPoint, len(spMounts))
+	for i, m := range spMounts {
+		var mp MountPoint
+		if m.Type != "bind" {
+			continue
+		}
```

Comment:
This will cause the `mountpoints` will always be `len(spMounts)` and some zero-value in the middle (i.e. when `continue`d). Is that expected? If not we can use `mountpoints := make([]MountPoint, 0, len(spMounts))` and `mountpoints = append(mountpoints, mp)` at line 356.

Reply:
Fixed. Thanks